docs/docs/references/configuration/cli/trivy_kubernetes.md

```diff
@@ -42,7 +42,7 @@ trivy kubernetes [flags] { cluster | all | specific resources like kubectl. eg:
       --download-java-db-only             download/update Java index database but don't run a scan
       --exit-code int                     specify exit code when any security issues are found
       --file-patterns strings             specify config file patterns
-  -f, --format string                     format (table, json, template, sarif, cyclonedx, spdx, spdx-json, github, cosign-vuln) (default "table")
+  -f, --format string                     format (table, json, cyclonedx) (default "table")
```

Comment:
should there be more content in the docs? Or ist that something I can/should work on?

Reply:
correct, I'll add some more info on feature and you could go over it

Reply:
> should there be more content in the docs? Or ist that something I can/should work on?

correct, I'll add some more info on feature and you could go over it